src/ledger/ledger-radix-babylon.ts

```diff
@@ -0,0 +1,87 @@
+import Transport from '@ledgerhq/hw-transport'
+import { errAsync, ok, ResultAsync } from 'neverthrow'
+import { LedgerPublicKeyRequest } from './schemas'
+
+export const encodeBip32 = (hdPath: string) => {
+  const path = hdPath.slice(2).split('/')
+  const length = `00${(path.length & 0xff).toString(16)}`.slice(-2)
+
+  const parts = path
+    .map((value) => 0x80000000 + parseInt(value, 10))
+    .map((value) => value.toString(16))
+    .join('')
+
+  return `${length}${parts}`
+}
+
+export enum LedgerErrorResponse {
+  UnlockDevice = '5515',
+  OpenRadixApp = 28161,
+}
+
+export enum LedgerInstructionCode {
```

Comment:
we're not using enums as a code convention. Lets make a decision before introduction them